### DISCUSSION TITLE
RTSP packets containing single partitions

### DISCUSSION THREAD

**Discussion by uzername123** [bluenviron/mediamtx#2327] [Questions]
Hi All 

sending stream from jetson nano with nvv4l2h264enc from camera

**gst-launch-1.0 nvv4l2camerasrc ! .....! nvv4l2h264enc ! rtspclientsink rtsp://localhost:port/streamname**

all goes ok

when change encoder to vp8 (nvv4l2vp8enc) - mediamtx receives stream but gives error:

**_2023/09/11 20:23:26 WAR [RTSP] [session d78fffd3] packets containing single partitions are not supported_**

what I could avoid such error?

Thnx

**Comment by aler9** [comment]
This is a bug with nvv4l2vp8enc, open an issue and provide a network dump by following instructions.
